saltstack/base/salt/luks/init.sls

```diff
@@ -0,0 +1,98 @@
+/etc/luks:
+  file.directory:
+    - name: /etc/luks
+    - user: root
+    - group: root
+    - mode: 700
+
+/etc/luks/bin:
+  file.directory:
+    - name: /etc/luks/bin
+    - user: root
+    - group: root
+    - mode: 700
+
+/etc/luks/backup:
+  file.directory:
+    - name: /etc/luks/backup
+    - user: root
+    - group: root
+    - mode: 700
+
+luks_backing_file:
+  file.managed:
+    - name: /etc/luks/luks
+    - user: root
+    - group: root
+    - mode: 600
+    - create: true
+
+passphrase_ciphertext_file:
+  file.managed:
+    - name: /etc/luks/passphrase_ciphertext
+    - user: root
+    - group: root
+    - mode: 600
+    - create: true
+
+passphrase_encryption_key_file:
+  file.managed:
+    - name: /etc/luks/passphrase_encryption_key
+    - user: root
+    - group: root
+    - mode: 600
+    - create: true
```

Comment:
For directories it makes sense, but what is the benefit of creating these zero-byte placeholders here? They will be prepared and overwritten by `create-luks-volume.sh` anyway.

Reply:
I used these as a way of enforcing 600 permissions.

Reply:
But, yeah it probably makes more sense to create these in the script just before they are needed.